libcudacxx/include/cuda/std/__cccl/exceptions.h

```diff
@@ -73,7 +67,7 @@
 struct __cccl_catch_any_lvalue
 {
   template <class _Tp>
-  _CCCL_HOST_DEVICE operator _Tp&() const noexcept;
```

Comment:
II am not sure if this will work if the exception is not bound by `const&` but only `&`